chat/uikit/ios.md

```diff
@@ -0,0 +1,386 @@
+self.navigationController?.pushViewController(conversationViewController, animated: true)
+```
+
+## Customizations
+`SKYChatConversationView` can be customized via
+
+1. setting properties of
+- `SKYChatConversationView.UICustomization()`
+- `SKYChatUIModelCustomization.default()`
+- `SKYChatConversationViewController`
+2. implementing delegates.
+
+
+## UICustomization
+You can customize `SKYChatConversationView` properties in the following way.
+
+```obj-c
+[SKYChatConversationView UICustomization].messageSenderTextColor = [UIColor blackColor];
+[SKYChatConversationView UICustomization].avatarHiddenForIncomingMessages = NO;
+```
+
+```swift
+SKYChatConversationView.UICustomization().messageSenderTextColor = UIColor.black
+SKYChatConversationView.UICustomization().avatarHiddenForIncomingMessages = false
+```
+
+The following properties are supported.
+### Avatar
+|Attribute|`Description`|Type|
+|---------|-----------|------|
+|`avatarBackgroundColor`|Background color of the avatar|Color|
+|`avatarTextColor`|Text color of the avatar|Color|
+|`userAvatarType`|Avatar Type, either showing initial of the username field or image|`SKYChatConversationViewUserAvatarType`, "initial" or "image"|
+|`avatarHiddenForIncomingMessages`|Hide incoming message avatar if true|Boolean, default False|
+|`avatarHiddenForOutgoingMessages`|Hide outgoing message avatar if true|Boolean, default True|
+
+### Status Text & Date Time Format
+|Attribute|`Description`|Type|
+|---------|-----------|------|
+|`messageStatusDelivering`|Delivering text|String, default "Delivering"|
+|`messageStatusDelivered`|Delivered text|String, default "Delivered"|
+|`messageStatusSomeRead`|Some read text|String, default "Some Read"|
+|`messageStatusAllRead`|All read text|String, default "All Read"|
+|`messageSentFailed`|Failed text|String, default "Failed"|
+|`messageDateFormatter`|Message delivery time date formatter|`DateFormatter`|
+
+### Title
+|Attribute|`Description`|Type|
+|---------|-----------|------|
+|`titleDisplayType`|See below|`SKYChatConversationViewTitleOptions`|
+
+
+If `default` is set, then the title bar displays `title` field value of the conversation record. If  `otherParticipants` is set,
+then the title bar displays participants' usernames (excluding the current user) in the title bar.
+
+```obj-c
+[SKYChatConversationView UICustomization].titleDisplayType = SKYChatConversationViewTitleOptionsOtherParticipants;
+```
+
+```swift
+SKYChatConversationView.UICustomization().titleDisplayType = .otherParticipants
+```
+
+
+### Message
+|`attribute`|description|type|
+|---------|-----------|------|
+|`messageSenderTextColor`|text color of the message sender|Color|
+
+
+## SKYChatUIModelCustomization
+You can change `userNameField` and `userAvatarField` via `SKYChatUIModelCustomization.default()`.
+
+
+```obj-c
+[[[SKYChatUIModelCustomization default]
+    updateWithUserNameField:@"name"]
+    updateWithUserAvatarField:@"profile_pic"];
```

Comment:
I think it is not common to chain the call this way in objective-c? Would be better to call the update separately like this?
```
[[SKYChatUIModelCustomization default] updateWithUserNameField:@"name"];
[[SKYChatUIModelCustomization default] updateWithUserAvatarField:@"profile_pic"];
```